llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -242,21 +243,15 @@ static void collectKernelModuleMap(
     KernelMapEntryScope EntryScope) {
 
   for (auto &F : M.functions()) {
-    if (F.getCallingConv() == CallingConv::SPIR_KERNEL) {
+    // Process module entry points: kernels and SYCL_EXTERNAL functions.
```

Comment:
Indirectly callable functions should also be kept. Would be nice (nit) to have a test on this.

Reply:
Added.